docs/flow-logic/merging.md

```diff
@@ -8,7 +8,7 @@ description: Merge data streams in you n8n workflows.
 
 Merging allows you to bring multiple data streams together.
 
-Merging data from different branches or nodes uses the [Merge](/integrations/builtin/core-nodes/n8n-nodes-base.merge/) node. To merge data from multiple node executions, use the [Code](/integrations/builtin/core-nodes/n8n-nodes-base.code/) node.
+Merging data from different branches or nodes uses the [Merge](/integrations/builtin/core-nodes/n8n-nodes-base.merge/) node. You can also use the [Code](/integrations/builtin/core-nodes/n8n-nodes-base.code/) node to merge data from multiple node executions. 
```

Comment:
@ayatnkw do you think it's worth adding text here about which versions allow you to do what, similar to what you did in the Merge node itself?

Reply:
good point, will add this!

Reply:
@ayatnkw I have a question on the current wording:
>Merging data from different branches or nodes uses the [Merge](/integrations/builtin/core-nodes/n8n-nodes-base.merge/) node. To merge data from multiple node executions in a single node or multiple nodes, use the [Code](/integrations/builtin/core-nodes/n8n-nodes-base.code/) node.

As a somewhat dumb user, I'm not sure this explains to me when I'd want to use the Code node vs. the Merge node. I'm guessing in many cases the Merge node would be sufficient, and the Code node just might be applicable in certain edge cases...could you confirm/clarify?

Reply:
Yes you're right. This is clarified in the sections further down in the page but I'll try to make this part clearer.